Engine/Accessors/TaskAccessor.cs

```diff
@@ -151,7 +151,7 @@ public static Type GetTaskResultType(Type taskType)
             }
             else
             {
-                throw new Exception();
+                throw new Exception($"Failed to match type [{taskType.FullName}] as a known 'Task' type.");
```

Comment:
@ppittle, let's put the `$"{taskType}"` into the string - that will also expand the generic arguments (if any).